### PR TITLE
#5515 Disable to execute start/stop requests concurrently

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-agent/src/main/scala/oharastream/ohara/agent/k8s/K8SClient.scala [hunk 1]
```diff
@@ -366,16 +366,15 @@ object K8SClient {
         private[this] def removePod(name: String, isForce: Boolean)(
           implicit executionContext: ExecutionContext
         ): Future[Unit] = {
-          val isForceRemovePod: String = if (isForce) "?gracePeriodSeconds=0" else ""
+          val gracePeriodSeconds = if (isForce) 0 else 2
```
Comment (chia7712) [inline ohara-agent/src/main/scala/oharastream/ohara/agent/k8s/K8SClient.scala:369]: 我把這部分先調低以避免可能的等待

0.12版本我們會將timeout變成APIs的一部分

Comment (jackyoh) [reply ohara-agent/src/main/scala/oharastream/ohara/agent/k8s/K8SClient.scala:369]: 預設 30 秒的等太久了。

Comment (chia7712) [reply ohara-agent/src/main/scala/oharastream/ohara/agent/k8s/K8SClient.scala:369]: 如果資料很多的話，搞不好要等更久

還是要把timeout變成參數比較有意義

**Comment by chia7712** [comment]
retry connectorIT

**Comment by chia7712** [comment]
retry otherIT

**Comment by chia7712** [comment]
retry streamIT

**Comment by chia7712** [comment]
retry collieIT

**Comment by jackyoh** [comment]
retry otherIT

**Comment by chia7712** [comment]
retry clientIT

**Pull request merged**